### ISSUE TITLE
Popped out panel seems to not refresh

### ISSUE THREAD

**Issue by lrargerich** [falcon71/kln90b#3]
When I pop out the KLN90 screen it quickly starts to look bad as if the screen was overdrawing over itself and the light bleeding around the cahracters.
Of course this can be me a complete noob at modding aircraft but just in case can you check if the popped out panel is working ok?
Thanks!

**Comment by lrargerich** [comment]
Ok I don't think it is a refresh problem but here is what I found. 
For some reason, the KLN90 panel, when popped out has some "scratches" (?) or places where light from the cockpit bleeds so depending on where I look in the cockpit the way the popped out KLN looks changes. 
![bleed](https://github.com/falcon71/kln90b/assets/7193221/55d394eb-3966-45c1-bc5b-6a8bf1490341)

**Comment by falcon71** [comment]
I hope you don't have a refresh problem.
I am however aware of the texture bleeding in the popped out window. It occurred when I added a glow effect to the screen. It looks very good in the 3D model, but quite bad in the popped out window. I'm not sure why it looks so bad here, somehow it seems to be rendered differently than the 3D one. Maybe it's a MSFS bug?
Do people actually use the popped out windows? I've never used them, so it never really bothered me. Maybe I will introduce a setting where you can decide if you want a nice 3D view with glow or a nice popped out view without glow.

Anyway, if you are feeling adventurous, you can modify the KLN90B.css file and delete all lines that start with filter:. This will remove the glow effect and should make it clear in the popped out window.

**Comment by lrargerich** [comment]
Removing the filter lines from the CSS file worked perfectly. For cockpit builders popped out windows are fundamental because we usually fly with only the windshield in the main screen and the instruments popped out to tablets using spacedesk or similar.

**Comment by functoid** [comment]
Even with glow disabled, the texture (which looks to be the main sim display) bleeds through around the anti-aliased edges of the text. It seems that if `background-color` is not set on the `documentElement`, the sim window bleeds through. I've noticed the same issue with some other instrument pop outs.

I tried a quick fix by adding `document.documentElement.style.backgroundColor = "black"` in `PageManager.setupContainer` (there's probably a better place to set this up) and it resolves the issue. See the before and after screen captures below.

![Image](https://github.com/user-attachments/assets/25a3d703-ca64-4d55-8ce5-7645a2fc98aa)
![Image](https://github.com/user-attachments/assets/fcb3545a-7d9d-4972-a419-dd0022ca3e22)
